styles/maxspeed.mapcss

```diff
@@ -836,17 +836,25 @@ node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:speed_limi
 }
 
 /* West German branch line speed signals (Lf 4 DS 301) */
-node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:speed_limit_distant"="DE-ESO:db:lf4"]["railway:signal:speed_limit_distant:form"=sign]["railway:signal:speed_limit_distant:speed"=~/^[1-8]0$/]
+node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:speed_limit_distant"="DE-ESO:db:lf4"]["railway:signal:speed_limit_distant:form"=sign]["railway:signal:speed_limit_distant:speed"=~/^([1-8][05])|0$/]
```

Comment:
The regular expression is syntactically wrong because it misses the round parantheses around the alternation (the options separated by `|`). It works but it relies on a undefined behaviour. Therefore, the regular expression should be rewritten as `/^([2-8]0|1[05]|0)$/`.